test/Test/Time/Property.hs

```diff
@@ -70,3 +91,11 @@ prop_readShowUnit = property $ do
     ratN <- forAll getRationalNum
     n    <- forAll getNumber0'9
     verifyAnyTime $ unitChooser n ratN
+
+prop_toUnit :: Property
+prop_toUnit = property $ do
+    ratN1 <- forAll getRationalNum
+    n1    <- forAll getNumber0'9
```

Comment:
It's better to move

```haskell
     ratN <- forAll getRationalNum
     n    <- forAll getNumber0'9
    -- and unitChooser here as well
```

into separate generator, like `anyTime` since you use it already three times. Better to remove code duplication.

Also, by coding convention, it's better to not add `get` or `gen` prefixes. I see that in `hedgehog` library itself all generators don't have `getPrefix`. So maybe it's better if we follow this library naming convention in out tests as well.